docs/hadoop-catalog.md

```diff
@@ -47,9 +63,13 @@ The Hadoop catalog supports creating, updating, deleting, and listing schema.
 
 ### Schema properties
 
-| Property name | Description                                    | Default value | Required | Since Version |
-|---------------|------------------------------------------------|---------------|----------|---------------|
-| `location`    | The storage location managed by Hadoop schema. | (none)        | No       | 0.5.0         |
+| Property name                                      | Description                                                                                    | Default value | Required                                                    | Since Version   |
+|----------------------------------------------------|------------------------------------------------------------------------------------------------|---------------|-------------------------------------------------------------|-----------------|
+| `location`                                         | The storage location managed by Hadoop schema.                                                 | (none)        | No                                                          | 0.5.0           |
+| `authentication.type`                              | The type of authentication for Hadoop catalog, currently we only support `kerberos`, `simple`. | `simple`      | No                                                          | 0.6.0           |
+| `authentication.impersonation-enable`              | Whether to enable impersonation for the Hadoop catalog.                                        | `false`       | No                                                          | 0.6.0           |
+| `authentication.kerberos.principal`                | The principal of the Kerberos authentication                                                   | (none)        | required if the value of `authentication.type` is Kerberos. | 0.6.0           |
+| `authentication.kerberos.keytab-uri`               | The URI of The keytab for the Kerberos authentication.                                         | (none)        | required if the value of `authentication.type` is Kerberos. | 0.6.0           |
```

Comment:
Will it extend the parent value if is not set?

Reply:
Resolved